### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by liruochen1998**
finally find the correct/elegant way to extract info from togetherJS to populate header. Now there are two buttons, one for end turn, one for refreshing the user data. eg. if new users join in, click on refresh and the total users will be updated.

### CODE REVIEW

**Review by Jaega — DISMISSED** [review]
Honestly I don't know how this works but looks good lol

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    names.push(myName);
+    myPeers.forEach(p => {
+      names.push(p.name);
+    });
+    this.setState({users: names});
+  }
+
+  endTurn() { 
+    this.setState({
+      currNum: (this.state.currNum+1)%3
```
Comment (liruochen1998) [inline src/components/Header/Header.js]: realized that I need to change 3

Comment (carolinepar) [reply src/components/Header/Header.js]: Is this assuming there are three people on the page?

Comment (liruochen1998) [reply src/components/Header/Header.js]: yes, that's from developing, but I just updated that to a variable

**Comment by carolinepar** [comment]
When I took a look it seems that the header is in an awkward place. Could you maybe move it to the top right corner above the log table?
I also don't see any name showing up for me. It just says `It's 's turn`.
Love the refresh button, I'm glad you found an elegant solution :D

**Comment by liruochen1998** [comment]
> When I took a look it seems that the header is in an awkward place. Could you maybe move it to the top right corner above the log table?
> I also don't see any name showing up for me. It just says `It's 's turn`.
> Love the refresh button, I'm glad you found an elegant solution :D

is showing up `it's 's turn` after you press refresh? Weird.. works on mine even when there's no other person joining. About the position-- yeah I think that might look better, I can change that. I make it here simply because it is on the top in our figma design. @carolinepar

**Comment by carolinepar** [comment]
> > When I took a look it seems that the header is in an awkward place. Could you maybe move it to the top right corner above the log table?
> > I also don't see any name showing up for me. It just says `It's 's turn`.
> > Love the refresh button, I'm glad you found an elegant solution :D
> 
> is showing up `it's 's turn` after you press refresh? Weird.. works on mine even when there's no other person joining. About the position-- yeah I think that might look better, I can change that. I make it here simply because it is on the top in our figma design. @carolinepar

Yeah it still doesn't work for me when I just press refresh. If I change my name and then press refresh it does work however. So maybe the initialization is missing something?

**Comment by carolinepar** [comment]
The header on the side looks great! Could you add a little spacing between the two buttons? They're a little on top of each other. After that and the merge conflict is handled, it'll be good to merge!

**Comment by liruochen1998** [comment]
just solved the conflict, I would say for the position of the button, don't worry about them for now, and we'll definitely make a lot of UI change in details later when other components coming in.

**Review by carolinepar — APPROVED** [review]

**Review by Jaega — APPROVED** [review]

**Pull request merged**